src/org/traccar/reports/Events.java

```diff
@@ -92,7 +92,7 @@ public static void getExcel(OutputStream outputStream,
             deviceEvents.setDeviceName(device.getName());
             sheetNames.add(WorkbookUtil.createSafeSheetName(deviceEvents.getDeviceName()));
             if (device.getGroupId() != 0) {
-                Group group = Context.getDeviceManager().getGroupById(device.getGroupId());
+                Group group = (Group) Context.getGroupsManager().getById(device.getGroupId());
```

Comment:
It will. But it might look longer
````java
Group group = (Group) Context.getGroupsManager().getById(device.getGroupId());
````
vs
````java
Group group = Context.getGroupsManager().getById(Group.class, device.getGroupId());
````